src/web/assets/js/photoGallery.js

```diff
@@ -0,0 +1,102 @@
+// photoGallery from http://ashleydw.github.io/lightbox/
+
+//links to/from this file:
+    //layout.html MODIFIED (head script & body script),
+    //index.html MODIFIED (script link to this file)
+    //ekko-lightbox.css ADDED (also adds some custom tags - photoGallery)
+
+(function() {
+    'use strict';
+    var numOfPhotos = 2; //USER DEFINED - photos per "gallery," ** auto updates bs column width below
+    var photos = [
+        [
```

Comment:
Thats a good idea. Originally I just had the images in an single-item array. I added the description later and just threw it into the array. Using and object makes sense. I will implement this.